From 889485db762ab14a18c80cd5be66720514d6cf6e Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Sat, 27 Jun 2020 15:55:27 -0700
Subject: [PATCH] include missing endian.h header

musl needs this as otherwise both __BYTE_ORDER and __BIG_ENDIAN are undefined.
---
 src/cap-ng.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cap-ng.c b/src/cap-ng.c
index 97a3dbd..7aa1029 100644
--- a/src/cap-ng.c
+++ b/src/cap-ng.c
@@ -33,6 +33,7 @@
 #include <sys/stat.h>
 #include <errno.h>
 #include <fcntl.h>
+#include <endian.h>
 #include <byteswap.h>
 #ifdef HAVE_PTHREAD_H
 #include <pthread.h>	// For pthread_atfork
